config.py

```diff
@@ -0,0 +1,6 @@
+# Please config the app display name here. 
```

Comment:
Maybe we can be explicit that the _values_ are the ones that should be changed to their respective names